_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -3090,6 +3090,7 @@ mfxStatus VideoVPPHW::VppFrameCheck(
             }
         }
 
+        MFX_CHECK_NULL_PTR1(pWorkOutSurf);
```

Comment:
Why it is not inside ```AssignTask```?

Reply:
Moved this check inside `AssignTask` function